src/Form/Type/UserType.php

```diff
@@ -46,7 +46,6 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $occupationChoices = (array) array_combine(self::DEFAULT_OCCUPATIONS, self::DEFAULT_OCCUPATIONS);
         $occupationChoices += ['Autre :' => '-'];
         $builder
-            ->add('identificationNumber', TextType::class)
```

Comment:
we must keep it, if the user puts his email instead of the number

Reply:
Well It's already in the form, but only for user creation. Do we need it on both?